### DISCUSSION TITLE
Purge Line

### DISCUSSION THREAD

**Discussion by kfinkbiner** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#21] [General]
First I want to state that I love this profile.  It has been working flawlessly for me with PLA and for that I thank you.  One question, I noticed you removed the purge line.  Is there a way to add it back, or could you tell me what gcode to add.  Thanks.

**Comment by suchmememanyskill** [comment]
How so? I'm currently using the brim as a purge line.

It got changed here: https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/commit/44f37f8b1099d1efebe27e4115e201d22445e095
